app.rb

```diff
@@ -0,0 +1,15 @@
+require 'sinatra/base'
+require 'sinatra/reloader'
+
+class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get '/' do 
+  'Hello World'
```

Comment:
I have seen many ppl do erb: index here to maintain separation of responsibilities.